pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.apache.mesos</groupId>
     <artifactId>hadoop-mesos</artifactId>
-    <version>0.1.0</version>
+    <version>0.1.1-SNAPSHOT</version>
```

Comment:
Actually, would you mind removing the version bump from this PR? We'll bump the version in master as there's another PR to be released, too.